doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -515,6 +523,7 @@ The absence of an operand for conditions which accept them simply requires that
 a value exists (e.g. the content of the header is not an empty string) for the
 condition to be considered true.
 
+
```

Comment:
are these whitespace and `:wq` changes intentional?

Reply:
Corrected